src/components/grid-aware/StatsBlock/StatsBlock.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./StatsBlock.module.css";
+
+const StatCards = PropTypes.shape({
+  number: PropTypes.string.isRequired,
+  statement: PropTypes.string.isRequired,
+});
+
+const StatCard = ({ number, statement }) => (
+  <div className={s.statCard}>
+    <div className={s.number}>{number}</div>
+    <div className={s.statement}>{statement}</div>
+  </div>
+);
+
+StatCard.propTypes = {
+  number: PropTypes.string.isRequired,
+  statement: PropTypes.string.isRequired,
+};
+
+const StatsBlock = ({ title, statCards }) => (
+  <div className={s.bleedWrapper}>
+    <div className={s.bleedMainContent}>
+      <section className={s.gridParent}>
+        <h1 className={s.title}>{title}</h1>
+        <div className={s.gridAreaBottom}>
+          {statCards.map((obj) => (
+            <div key={obj.number}>
```

Comment:
This `<div>` doesn't serve any purpose, so you can remove it. You'll still need to move the `key={object.number}` to the `<StatCard>`, though, since in React, we always have to set the [`key`](https://reactjs.org/docs/lists-and-keys.html#keys) special prop in the body of a `.map()`.